### ISSUE TITLE
How to use SmolVLA to reproduce the same result with the official report?

### ISSUE THREAD

**Issue by lzy121121** [huggingface/lerobot#3628]
❓ Technical Question

### Environment & System Info

```Shell
- LeRobot version: 0.5.2
- Platform: Linux-5.15.0-160-generic-x86_64-with-glibc2.39
- Python version: 3.12.12
- Huggingface Hub version: 1.14.0
- Transformers version: 5.5.4
- Datasets version: 4.8.5
- Numpy version: 2.2.6
- FFmpeg version: 6.1.1-3ubuntu5
- PyTorch version: 2.11.0+cu128
- Torchcodec version: 0.11.1+cpu
- Is PyTorch built with CUDA support?: True
- Cuda version: 12.8
- GPU model: NVIDIA A100-SXM4-80GB
- Using GPU in script?: <fill in>
- lerobot scripts: ['lerobot-calibrate', 'lerobot-dataset-viz', 'lerobot-edit-dataset', 'lerobot-eval', 'lerobot-find-cameras', 'lerobot-find-joint-limits', 'lerobot-find-port', 'lerobot-imgtransform-viz', 'lerobot-info', 'lerobot-record', 'lerobot-replay', 'lerobot-rollout', 'lerobot-setup-can', 'lerobot-setup-motors', 'lerobot-teleoperate', 'lerobot-train', 'lerobot-train-tokenizer']
```

### Description

I used Lerobot to reproduce the official results of SmolVLA as shown in https://huggingface.co/spaces/HuggingFaceVLA/libero-vla-leaderboard

My command is 
`uv run lerobot-eval  --policy.path=HuggingFaceVLA/smolvla_libero  --env.type=libero  --env.task=libero_goal  --eval.batch_size=1  --eval.n_episodes=10 --env.max_parallel_tasks=1`
And finally I got this result, which differs from the official result significantly. Does anybody know the correct eval config or what's wrong with my setup? 
<img width="2679" height="838" alt="Image" src="https://github.com/user-attachments/assets/074e69d7-3e3f-4f7e-b43e-3a35ab54391b" />

Thanks for your help!

**Comment by HueXiang** [comment]
hi, may i ask if the problem has been solved?
recently i tried evaluating libero using a dinov2 + google t5 + flow matching model
i have high SR (>80) on all suites except for libero goal (<40)
i now tried diff config but cant raise the SR.
thank you!

**Comment by captainfffsama** [comment]
It looks like SmolVLA v0.4.4 performs better than v0.5.2. I used the same data and identical training settings, but the final loss with v0.4.4 was about 33% lower than with v0.5.2.
